recipes-kernel/linux/files/patches-6.1/0097--arm64-dts-ti-iot2050-Disable-lock-step-for-all-iot20.patch

```diff
@@ -0,0 +1,64 @@
+From 1236b3418b6566614d3b5f12626f007541399959 Mon Sep 17 00:00:00 2001
+From: Li Hua Qian <huaqian.li@siemens.com>
+Date: Fri, 22 Mar 2024 09:36:47 +0800
+Subject: [PATCH] arm64: dts: ti: iot2050: Disable lock-step for all iot2050
+ boards
+
+The PG1 A variant of the iot2050 series has been identified which
+partially lacks support for lock-step mode. This implies that all
+iot2050 boards can't support this mode. As a result, lock-step mode has
+been disabled across all iot2050 boards for consistency and to avoid
+potential issues.
+
+Signed-off-by: Li Hua Qian <huaqian.li@siemens.com>
+---
+ arch/arm64/boot/dts/ti/k3-am65-iot2050-common-pg2.dtsi | 5 -----
+ arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi     | 5 +++++
+ arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic.dts     | 5 -----
```

Comment:
You forgot to clean up also k3-am6548-iot2050-advanced-m2.dts.

Reply:
It was already cleaned up in https://github.com/siemens/meta-iot2050/commit/34b7ef3c9af6a1abb3d36cf9c0a8822741577117.

Reply:
Oh, indeed, was looking at stale version of the kernel here.